### ISSUE TITLE
POW broken

### ISSUE THREAD

**Issue by craig** [DropMorePackets/berghain#5]
After performing a few requests, the POW does show up. However, it cannot be solved by google-chrome-stable 121.0.6167.184-1:

![pow](https://github.com/DropMorePackets/berghain/assets/99729/1b66c5bd-2145-4cbf-8edd-95b8ffb7e345)

On Firefox 115.8.0esr-1~deb11u1, the first solve works (nice animation!) . After performing some more requests, I'm seeing the POW again, but something is broken:

![pow2](https://github.com/DropMorePackets/berghain/assets/99729/4c078595-4d4f-4dbc-9e68-fda649427d8e)

**Comment by fionera** [comment]
Currently berghain only works when connecting via HTTPS, as these browser APIs are not available without a encrypted connection

**Comment by fionera** [comment]
I made a branch which should fix this by adding a fallback to a js implementation of sha256 but at the cost of filesize:
```
Without sha.js:
dist/index.html  6.48 kB │ gzip: 2.86 kB

With sha.js
dist/index.html  19.80 kB │ gzip: 7.68 kB
```

I could add both to the routing and return them conditionally, so that a HTTPS response always gets the smaller file and HTTP gets the bigger file. Don't know if that worth the effort as you have to increase the buffer-size to >20kb to fit the new file

**Comment by fionera** [comment]
Fixed by #14
